### PR TITLE
add loadSecureUrls props to resolve mixed content issue

### PR DESCRIPTION
**Pull request by sndsabin**
add `onlySecure` props
-  if `true`, loads only secure ( `https://` ) resources. If no secure resource was found, then `` and `<video>` element wont be rendered
- default `false` works as before

### With `onlySecure={false}`

**works exactly as before**

Screenshot from 2020-03-25 00-30-56

Screenshot from 2020-03-25 00-31-25

### With `onlySecure={true}`

**No `video` tag present in DOM**

Screenshot from 2020-03-25 00-31-45

Screenshot from 2020-03-25 00-32-20

**P.S: Always loads resources over `https` whenever possible.**

### CODE REVIEW

File: src/components/CardMedia.tsx [hunk 1]
```diff
@@ -1,48 +1,77 @@
...
+
+  return null
+
+}
+
+const VideoWrapper = ({data, secureVideoUrl, onlySecure, autoPlay}) => {
+  if (onlySecure && !secureVideoUrl) return null
+  const videoUrl = data.video && ((onlySecure && secureVideoUrl) ? secureVideoUrl : data.video[0])
+
+  if (videoUrl) {
```
Comment (winhtaikaung) [inline src/components/CardMedia.tsx]: Can we do like this ?

```
if (!videoUrl) {
    return null
  }
  return  <Video
            muted
            onCanPlayThrough={e => {
              let video = e.target
              setTimeout(() => {
                video.pause()
              }, 1000)
            }}
            autoPlay={autoPlay}
            src={videoUrl}
          />
```

File: README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ import { ReactTinyLink } from 'react-tiny-link'
 | **defaultMedia** | Default value to provide the media for failure cases      | string                          | N.A                                            | `false`  |
 | **onError**      | Error callback on when the url failed to fetch            | onError(`error:Error`)          | N.A                                            | `false`  |
 | **onSuccess**    | Success callback on when the url was fetched successfully | onSuccess(`data:IResponseData`) | N.A                                            | `false`  |
+| **onlySecure**    | Load only secure ( `https://` ) resources. If no secure resource was found, then don't render the `<img>` and `<video>` element  | boolean | default(`false`)                                            | `false`  |
```
Comment (winhtaikaung) [inline README.md]: Shall we name it to `loadSecureUrls` ?

Comment (sndsabin) [reply README.md]: or `loadOnlySecureUrls` maybe?

File: src/components/CardMedia.tsx (hunk 1 above)
Comment (sndsabin) [reply src/components/CardMedia.tsx]: done

File: README.md (hunk 2 above)
Comment (sndsabin) [reply README.md]: renamed `onlySecure` props to `loadSecureUrls`

Comment (winhtaikaung) [reply README.md]: Hi @sndsabin I think i typed wrong prop name actually it should be singular `loadSecureUrl`

**Comment by winhtaikaung** [comment]
Hi @sndsabin  I 've created a branch for your changes with the name of `sndsabin/master` please checkout from it and send  PR to that `sndsabin/master` branch again. :pray: 
Sorry for inconvenicence

**Comment by sndsabin** [comment]
@winhtaikaung a new PR is open for this same changes https://github.com/winhtaikaung/react-tiny-link/pull/46

**Pull request closed without merging**